tests/burstedde_bfbt.prm

```diff
@@ -0,0 +1,105 @@
+# The 3D polynomial Stokes (burstedde) benchmark, for which an
```

Comment:
can you change the comment to read "test burstedde benchmark using BFBT preconditioner", remove all the other comments here, include the burstedde.prm from the benchmarks folder and only include the relevant sections here (mesh refinement and solver settings).